js/imageeditor/shared/core/editor.ts

```diff
@@ -295,20 +296,24 @@ export class LayerManager {
 		width,
 		height,
 		layer_name,
-		user_created
+		user_created,
+		layer_id = undefined,
+		make_active = false
```

Comment:
When a new layer is added via the UI, we want to set that as the active layer but when we are creating them programmatically we want to keep the previous 'active layer' as the active layer.